blocks/product-list/product-list.css

```diff
@@ -0,0 +1,150 @@
+.product-list ul {
+  --card-height: auto;
+
+  list-style: none;
+  margin: 0;
+  margin-top: var(--udexSpacer40);
+  padding: 0;
+  display: grid;
+  grid-template-columns: repeat(1,1fr);
```

Comment:
Minor thing: repeat(1,xyz) is "repeat xyz once", so this could also be written as grid-template-columns: 1fr;